Reference/Querying/IPublishedContent/index.md

```diff
@@ -13,10 +13,13 @@ To access the current page in your macros or templates, copy-paste the below Raz
 	<h1>@pageName</h1>
 
 ## [Properties](Properties.md)
-Listing and explanation of IPublishedContent properties & standard helpers for Content and Media.
+Listing and explanation of IPublishedContent properties, standard helpers for Content and Media.
```

Comment:
Not native English speaking, but if you add a comma you need you add "and extension methods". Otherwise the "IPC properties" are defined as "standard helpers for Content".